src/devices/sound/c140.cpp

```diff
@@ -100,15 +100,15 @@ c140_device::c140_device(const machine_config &mconfig, device_type type, const
 	, m_stream(nullptr)
 	, m_baserate(0)
 {
-	memset(m_REG, 0, sizeof(uint8_t)*0x200);
-	memset(m_pcmtbl, 0, sizeof(int16_t)*8);
```

Comment:
It's pointless if you don't use `std::begin`/`std::end` to find beginning/end.  Like this you've still got hard-coded sizes and haven't gained anything.